include/vcpkg/base/format.h

```diff
@@ -29,8 +29,8 @@ namespace vcpkg
 
 namespace fmt
 {
-    template<class Char>
-    struct formatter<vcpkg::LineInfo, Char>
+    template<>
```

Comment:
Technically these can be declared in fwd headers now.

Reply:
I don't believe that declaring these in fwd headers is useful, since you should never be passing around references to them.